src/api_proxy/service_control/request_info.h

```diff
@@ -141,18 +141,34 @@ enum ScResponseErrorType {
   CONSUMER_QUOTA = 5,
 };
 
+// Store the info of service control http call failure/call error response.
+struct ScResponseError {
+  std::string name;
```

Comment:
name is kind of confusing?   is this "message"?

Reply:
This is the error name of service control response, which will be the response code detail, like `SERVICE_NOT_ACTIVATED` or `UNAVAILABLE`.

Reply:
how about error_from_server?

Reply:
It can be either from SC server or it can be a network error. I'm ok with `name`, maybe `display_name`. This is essentially what gets exposed in the log entry.